src/components/FileUploader.tsx

```
@@ -1,96 +1,202 @@
+  setSelectedFiles
+}: FileUploaderProps) {
+  const [rejectedFiles, setRejectedFiles] = useState<File[]>([]);
+
+  // Create an accept object from the acceptedFileTypes array
+  const accept = acceptedFileTypes.reduce((acc, type) => {
+    acc[type] = [];
+    return acc;
+  }, {} as Record<string, string[]>);
+
+  const onDrop = useCallback(
+    (acceptedFiles: File[], rejected: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)